src/app/components/Application/Application.jsx

```diff
@@ -16,11 +16,7 @@ class Application extends React.Component {
   render() {
     return (
       <div className="app-wrapper">
-        <Header
-          skipNav={{ target: 'mainContent' }}
-          navData={navConfig.current}
-          urlType="absolute"
-        />
+        <Header skipNav={{ target: 'mainContent' }} navData={navConfig.current} urlType="absolute" />
```

Comment:
I think one of the linters also prefers having this as multiline.

Reply:
Changed all to multiline and added eslint rule